src/main/java/org/example/controllers/JobRoleController.java

```diff
@@ -53,25 +62,39 @@ public Response getAllJobRoles(
                     pageSize);
             int sanitisedPage = paginationSanitiser.sanitisePage(
                     page, sanitisedPageSize, totalRecords);
+            String sanitisedFieldName = orderBySanitiser
+                    .sanitiseFieldName(fieldName);
+            String sanitisedOrderBy = orderBySanitiser.sanitiseOrderBy(orderBy);
             List<JobRoleResponse> jobRoles = jobRoleService
-                    .getAllJobRoles(sanitisedPage, sanitisedPageSize);
+                    .getAllJobRoles(sanitisedPage, sanitisedPageSize,
+                            sanitisedFieldName, sanitisedOrderBy);
             Pagination pagination = new Pagination(
                     jobRoleService.getTotalpages(
                             sanitisedPageSize,
                             sanitisedPage),
                             sanitisedPage,
                    sanitisedPage + 1,
                 sanitisedPage - 1);
+            RoleOrdering roleOrdering = new RoleOrdering(
```

Comment:
Luke, just use the two values that you have here at lines 61 and 63.... then you can delete these two methods on the JobRoleService... they dont do anything else anyway, they just pass the string back out that you give it :)